assets/src/js/Components/ArchivePicker/blockUpdateFunctions.js

```diff
@@ -58,6 +58,7 @@ export const updateMediaAndTextAttributes = (image, currentBlock) => {
       mediaAlt: image.alt_text,
       mediaId: image.id,
       mediaType: 'image',
+      useFeaturedImage: false,
```

Comment:
Is it complex if we rename it? Basically because, as a React convention, `use` prefix is used only for `hooks`.

Reply:
That is how the WP attribute is named.